CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## UNRELEASED
 
+- Fix: Outbox producer would crash if no Deimos producer was defined for the topic.
```

Comment:
What is this scenario where no producer will be defined for the topic. Does outbox not require me to define "producer block" in Karafka.rb?

Reply:
In this case, the messages are being inserted by the old system but published by the new system (which doesn't have configurations for them).